app/components/views/TransactionsPage/HistoryTab/Page.js

```diff
@@ -40,7 +40,9 @@ const Page = ({
       <div className="history-content-title-text">
         <T id="history.title" m="Transaction History" />
       </div>
-      <div className="history-select-tx-types-area">
+      <div className="history-select-tx-types-area" style={transactions.length == 0
+        ? { display: "none" }
+        : { display: "initial" } } >
```

Comment:
We don't use inline style

Reply:
Well I suppose we do use inline styles in certain applications.  But I'd rather this be 

`style{{display: transactions.length == 0 ? "none" : "initial"}}`

Reply:
OK great think its ready now.